webtau-utils/src/test/groovy/com/twosigma/webtau/utils/StringUtilsTest.groovy

```diff
@@ -77,4 +77,16 @@ line #_3\r""")
         def number = StringUtils.convertToNumber("123,000")
         assert number == 123000
     }
+
+    @Test
+    void "should start with"() {
+        Assert.assertEquals("/foo", StringUtils.ensureStartsWith("/foo", "/"))
```

Comment:
should we use `assert` for consistency?

Reply:
There's actually a mixture in this test but `assert` seems more prevalent so I've switched it to that.